WallPaperDev/WallPaperDev/Extensions/UIView+Extension.swift

```diff
@@ -10,6 +10,7 @@ import Foundation
 import UIKit
 
 extension UIView {
+  // (Jackson) I don't think we'll be needing this anymore
```

Comment:
Any followup here before merging?  If we want to keep it around let's mark as a todo and keep yourself as the owner! 
`// TODO: (Jackson) I don't think we'll be needing this anymore`